grakn-test/src/test/java/ai/grakn/test/graql/hal/HALBuilderTest.java

```diff
@@ -0,0 +1,72 @@
+package ai.grakn.test.graql.hal;
+
+import ai.grakn.GraknGraph;
+import ai.grakn.graphs.AcademyGraph;
+import ai.grakn.graql.MatchQuery;
+import ai.grakn.graql.Query;
+import ai.grakn.test.GraphContext;
+import mjson.Json;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static ai.grakn.graql.internal.hal.HALBuilder.renderHALArrayData;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class HALBuilderTest {
+
+    @ClassRule
+    public static final GraphContext academyGraph = GraphContext.preLoad(AcademyGraph.get());
+
+    @Test
+    public void testHALStructure() {
```

Comment:
Sorry to nitpick. Could you try to follow the same convention as everyone else i.e. something like
```
@test
public void whenDoingThisSpecificThing_ThisHappens
```

It makes the tests more clear to the outsider. So for example `testHALStructure()` might be `whenReceivingHALResponseForMatchQuery_EnsureResponseContainsConceptDetails`